src/app/jobs/components/job-list/job-list.component.ts

```diff
@@ -71,4 +77,4 @@ export class JobListComponent implements OnInit, OnDestroy {
   ngOnDestroy(): void {
     this.polling.unsubscribe();
   }
-}
+}
```

Comment:
You also need to add a newline again in the end of the file, otherwise ng lint will cause the failure of Travis build